platforms/cosmwasm/src/platformUtils.ts

```diff
@@ -79,10 +85,31 @@ export module CosmwasmUtils {
       return BigInt(amount);
     }
 
-    const { amount } = await rpc.getBalance(walletAddress, token.toString());
+    const addrStr = new CosmwasmAddress(token).toString();
+    const { amount } = await rpc.getBalance(walletAddress, addrStr);
     return BigInt(amount);
   }
 
+  export async function getBalances(
+    chain: ChainName,
+    rpc: QueryClient & BankExtension & IbcExtension,
```

Comment:
Should we keep this one as the default `CosmWasmClient` and convert to the rpc type we need in the body of the fn?